.github/PULL_REQUEST_TEMPLATE/pull_request.md

```diff
@@ -0,0 +1,15 @@
+## Description
+<Provide a detailed explanation of the changes, why they were necessary, and their benefits or impact>
+<Explain the problem you are fixing or the feature you are adding>
+
+## Checklist
+- [ ] I have added test cases for this feature
+- [ ] I have updated the documentation (if necessary)
+- [ ] I have run and passed all tests
+- [ ] I have checked for code smells
```

Comment:
These are already covered by our pipeline. I don't think we need to manually check these items.

Reply:
They're covered by the pipeline but it's also a reminder to the contributor to ensure (?)